geonode/__init__.py

```diff
@@ -1,6 +1,7 @@
+# -*- coding: utf-8 -*-
 #########################################################################
 #
-# Copyright (C) 2012 OpenPlans
+# Copyright (C) 2016 OSGeo
```

Comment:
This is a code donation, the above should be:
````
# Copyright (C) 2012 Open Source Geospatial Foundation
````

Unless the file was modified in 2016:
````
# Copyright (C) 2012-2016 Open Source Geospatial Foundation
````